src/cli/telemetry/helpers.ts

```diff
@@ -1,28 +1,10 @@
 import { getCompilerSystem } from '../state/stencil-cli-config';
 
-interface TerminalInfo {
-  /**
```

Comment:
FWIW, I think these JSDocs were fine to leave in  - I don't have definitive plans, but I'd like us to move to a 'better' documented codebase ('better' is subjective and can take on a variety of forms 😉 )

Reply:
Brought em back!